Test.js

```diff
@@ -2,34 +2,45 @@
 /* eslint-disable no-undef */
 const fs = require("fs");
 
+const PROBLEMS_FOLDERS = [
+  "./LeetcodeProblems/Algorithms/easy/",
+  "./LeetcodeProblems/Algorithms/medium/",
+  "./LeetcodeProblems/Algorithms/hard/"
+];
+
 const TESTS_FOLDERS = [
   "./LeetcodeProblemsTests/Algorithms/easy/",
   "./LeetcodeProblemsTests/Algorithms/medium/",
   "./LeetcodeProblemsTests/Algorithms/hard/"
-]
+];
 
 const REGEX_PATTERN_HIDDEN_FILES = /(^|\/)\.[^\/\.]/g;
 
-var test_all = async function () {
+const get_all_tests = async function (paths) {
+  let problems = [];
+  for(const i in paths) {
+    const folder = paths[i];
+    const new_problems = await loadProblemsFiles(folder); // await
+    problems = problems.concat(new_problems);
+  }
+  return problems;
+};
+
+const test_all = async function () {
```

Comment:
Could you rename the following functions?

- `test_all` -> `runAllTests`
- `solve` -> `solveProblem`

Also, let's use `camelCase` everywhere. I noticed we have a mix. A linter rule could solve this too (not need to do it here).

Reply:
updated all variables and functions in this file to be `camelCase`
I also added the eslint rule to make it visible and noticed there are a lot of issues

I can create another PR to solve all of those linter issues

Reply:
Love it! that'd be amazing.